packages/mongo/upsert_compatibility_test.js

```diff
@@ -0,0 +1,153 @@
+if (Meteor.isServer) {
```

Comment:
I believe you could `api.addFiles` this test file only on the server platform, instead of wrapping the entire file with `if (Meteor.isServer) { ... }`.